src/styles/reset.css

```diff
@@ -0,0 +1,19 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, Helvetica, sans-serif;
+  font-weight: var(--font-weight-main);
+}
+
+body,
+article,
+p,
+a {
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
```

Comment:
`body` and `article` don't have default padding and text-decoration, `a` doesn't have default `margin`